std/container/package.d

```diff
@@ -168,7 +168,7 @@ array.linearRemove(array[].find(2).take(1));
 assert(array[].equal([1, 3]));
 ---
 
-When any $(LINK2 std_range.html, range) can be passed as an argument to
+When any $(REF_ALTTEXT range, std, range) can be passed as an argument to
```

Comment:
I was fearing so - is there any macro similar to REF_ALTTEXT that we could use here?